core/blockchain.go

```diff
@@ -77,6 +77,7 @@ const (
 	validatorStatsCacheLimit           = 1024
 	validatorListCacheLimit            = 10
 	validatorListByDelegatorCacheLimit = 1024
+	pendingCrossLinksCacheLimit        = 2
```

Comment:
should make this larger.

Reply:
actually 1 is enough, since there is one key and corresponding to a list.